app/views/users/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>User: <%= @user.username%></h1>
+<h1>Followers: <%= @user.followers.length %></h1>
+<ul>
+  <% @user.followers.each do |follower| %>
```

Comment:
@idprietop getting the followers needs to be done in the controller, the views should be only in charge of rendering templates